python/Ganga/Utility/execute.py

```diff
@@ -261,7 +276,9 @@ def execute(command,
             logger.error("Command: %s" % command)
             logger.error("stdout: %s" % stdout)
             logger.error("stderr: %s" % stderr)
-            raise Exception("Missing update env after running command")
+            # Too specific? Hopefully we'll never know
+            from Ganga.Core.exceptions import RuntimeException
```

Comment:
I saw a circular dependence when I tried to force this to be triggered and I'll rename then